packages/sprite-tiling/src/TilingSprite.ts

```diff
@@ -265,9 +263,9 @@ export class TilingSprite extends Sprite
      * @param {number} height - the height of the tiling sprite
      * @return {PIXI.TilingSprite} The newly created texture
      */
-    static from(source, width, height)
+    static from(source: TextureSource, options: ISize): TilingSprite
     {
-        return new TilingSprite(Texture.from(source), width, height);
+        return new TilingSprite(Texture.from(source), options.width, options.height);
```

Comment:
I think this is a better API, but you need to provide backward compatibility. For instance:
```js
// Deprecated
if (typeof options === 'number') {
  deprecation('5.3.0', 'TilingSprite.from use options instead of width and height args');
  options = { width: options, height: arguments[2] } as ISize;
}
```

Reply:
Ok i've added this. Do you guys think it would be a good idea to update the other `.from` methods in this class and deprecate them to keep it consistent? 

for example
`static fromFrame(frameId: string, width: number, height: number): TilingSprite`